src/App.js

```diff
@@ -17,58 +17,44 @@ const goodsFromServer = [
 class App extends React.Component {
   state = {
     selector: false,
-    close: false,
   };
 
   closeButton = (text) => {
     this.setState({ selector: false });
-    this.setState({ close: false });
-    document.querySelector('.App__list--active').className = 'App__list';
   }
 
-  writeSrelector = (event, value) => {
+  writeSelector = (event, value) => {
     this.setState({ selector: value });
-    this.setState({ close: true });
-    const key = event.target;
-
-    const active = document.querySelector('.App__list--active');
-
-    if (active !== null) {
-      active.className = 'App__list';
-    }
-
-    key.className = 'App__list App__list--active';
   }
 
   render() {
-    const { selector, close } = this.state;
+    const { selector } = this.state;
 
     return (
       <div className="App">
         <h1>
           Selected good: -
           {' '}
           { selector }
-          <span>
-            <button
-              type="button"
-              className={close && 'App__close'}
-              onClick={this.closeButton}
-            >
-              {close}
-            </button>
-          </span>
+          <button
+            type="button"
+            style={{
+              display: 'inline-block',
+            }}
+            className={selector && 'close'}
```

Comment:
class name should describe what it makes, `visible` is more meaningful in this case